Dockerfile

```diff
@@ -14,7 +14,12 @@ RUN yarn build
 WORKDIR /app
 RUN yarn
 RUN yarn build
+RUN ncc build lib/index.js -o dist
 
+FROM  node:18-alpine
+WORKDIR /app
+COPY --from=builder /app/dist/ dist/
+ENV NODE_ENV=production
 EXPOSE 9464
 
-CMD [ "yarn", "start" ]
```

Comment:
this was invoking `ts-node` even after running all the `yarn build`, kinda weird

Reply:
turns out the k8s command overrode this to use the built lib anyway 🤷‍♂️